bindings_node/src/conversations.rs

```diff
@@ -121,6 +121,20 @@ impl NapiConversations {
     ))
   }
 
+  #[napi]
+  pub fn find_dm_group_by_target_inbox_id(&self, target_inbox_id: String) -> Result<NapiGroup> {
```

Comment:
I try to avoid the word group in these bindings when it's referring to a "conversation" But this is probably fine since the node bindings need a major refactor to support dms. cc @rygine 

```suggestion
  pub fn find_dm_by_target_inbox_id(&self, target_inbox_id: String) -> Result<NapiGroup> {
```